### PR TITLE
iOS Implementation for FusionLocalAuth

### PR DESCRIPTION
**Pull request by 6vedant**

### CODE REVIEW

File: FusionLocalAuth/Sources/FusionLocalAuth_Common/LocalAuthManager.swift [hunk 1]
```diff
@@ -97,7 +139,8 @@ public protocol LocalAuthManagerProtocol {
      *        cancelText: String
      *
      */
-    func requestBiometricAuthentication(authTitle:String, reasonTitle:String, cancelText:String, status: @escaping (AuthError?) -> Void )
+    
+    func requestBiometricAuthentication(authTitle:String, reasonTitle:String, cancelTitle:String, completionStatus: @escaping (Bool, AuthError?) -> Void )
```
Comment (gr-markin) [inline FusionLocalAuth/Sources/FusionLocalAuth_Common/LocalAuthManager.swift:143]: Just remembered, I used the "on<Something>" notion for callbacks in my code before, like "onComplete", "onError", "onSuccess" etc., it's not a must, but if you like, you can use it also.

**Review by gr-markin — APPROVED** [review]
I don't merge it automatically in case you want to proceed in the same branch. If I should merge every PR, just let me know.

File: FusionLocalAuth/Sources/FusionLocalAuth_Common/LocalAuthManager.swift (hunk 1 above)
Comment (6vedant) [reply FusionLocalAuth/Sources/FusionLocalAuth_Common/LocalAuthManager.swift:143]: Sure will use it. Thanks for letting me know.

**Comment by 6vedant** [comment]
> I don't merge it automatically in case you want to proceed in the same branch. If I should merge every PR, just let me know.

@gr-markin I will take care of the merge. No worries.

**Pull request merged**